extensions/Groq/README.md

```diff
@@ -1,23 +1,24 @@
-This extension contains the Groq client wrapper to use with AIConfig:
+This extension contains the Groq client wrapper to use with AIConfig so you can use models supported by the [GroqAPI](https://console.groq.com/docs/quickstart).
 
-# Part 1: Register your API key
+## Step 1: Set your Groq API key
 
-Create an API key from the Groq [website](https://console.groq.com/keys) and save it in your environment under the name `GROQ_API_KEY`. We recommend saving it into your home `~/.env` file, or one of your home hidden files like `~/.bashrc` or `~/.zshrc` so that you don't need to redefine it every session whenever you open a new terminal:
+1. Create an API key from the Groq [website](https://console.groq.com/keys).
+2. Add the key to your home `~/env` file: `GROQ_API_KEY = '<your-api-key>'`
```

Comment:
`GROQ_API_KEY = '<your-api-key>'` shouldn't have spaces, make it `GROQ_API_KEY='<your-api-key>'`